File: api/src/main/java/com/wheats/api/mypage/dto/CreateSupportTicketRequest.java
package com.wheats.api.mypage.dto;

public class CreateSupportTicketRequest {

    private Long storeId;   // 어떤 가게에 대한 문의인지
    private String title;   // 문의 제목
    private String message; // 문의 내용

    public CreateSupportTicketRequest() {
        // JSON 역직렬화용 기본 생성자
    }

    // ===== Getter / Setter =====

    public Long getStoreId() {
        return storeId;
    }

    public void setStoreId(Long storeId) {
        this.storeId = storeId;
    }

    public String getTitle() {
        return title;
    }

    public void setTitle(String title) {
        this.title = title;
    }

    public String getMessage() {
        return message;
    }

    public void setMessage(String message) {
        this.message = message;
    }
}

File: mobile/app/src/main/java/com/example/mobile/data/model/CreateSupportTicketRequest.kt
package com.example.mobile.data.model

/**
 * 새 1:1 문의 작성 요청 DTO
 * - 서버의 CreateSupportTicketRequest(Java)와 필드 구조를 맞춘 버전
 */
data class CreateSupportTicketRequest(
    val storeId: Long, // 어떤 가게에 대한 문의인지
    val title: String, // 문의 제목
    val message: String // 문의 내용
)


